Park-Swift/bin/install

```diff
@@ -6,4 +6,8 @@ npm install @react-navigation/material-top-tabs react-native-tab-view
 
 npx expo install react-native-pager-view
 
-nvm install node --reinstall-packages-from=current
\ No newline at end of file
+nvm install node --reinstall-packages-from=current
+
+npm install react-native-keyboard-aware-scroll-view
```

Comment:
Not sure if this caused an issue, but I can't run the app from my branch. 
<img width="1186" alt="Screenshot 2024-01-28 at 10 14 45 PM" src="https://github.com/void-tech-um/park-swift/assets/91640709/6ef44f0a-73d0-4675-a095-cf45c1fbc535">
I'm also not sure how yarn got included in the build but we are using npm, not yarn for this project. They are both package managers that do similar things and should not be mixed. The installation from @ashmoney024 branch worked so not sure what went wrong here. Let me know if I missed something!

Reply:
It also kinda broke my npm. 
<img width="1192" alt="Screenshot 2024-01-28 at 10 21 12 PM" src="https://github.com/void-tech-um/park-swift/assets/91640709/ff0a2026-21f2-471f-b2b8-3788265af8ea">